src/statement.js

```diff
@@ -98,9 +98,9 @@ class Statement {
    * @return Object                   Returns `this`.
    */
   where(conditions) {
-    var conditions = Array.isArray(conditions) && arguments.length === 1 ? conditions : Array.prototype.slice.call(arguments);
-    if (conditions.length) {
-      this._parts.where.push(conditions);
+    const args = Array.isArray(conditions) && arguments.length === 1 ? conditions : Array.prototype.slice.call(arguments);
```

Comment:
I always try to avoid `[].slice.call(arguments)` due to performance concerns. Can we use rest arguments here?